Composer/packages/client/src/components/ProjectTree/ProjectTree.tsx

```diff
@@ -340,7 +340,7 @@ export const ProjectTree: React.FC<Props> = ({
     );
   };
 
-  const renderDialogHeader = (skillId: string, dialog: DialogInfo, depth: number) => {
+  const renderDialogHeader = (skillId: string, dialog: DialogInfo, padLeft: number) => {
```

Comment:
Nit: depth is data, but padLeft is rendering concern.  This makes the caller have to decide the padding rather than the rendering function being called.